lib/vm/src/vmoffsets.rs

```diff
@@ -53,6 +53,8 @@ pub struct VMOffsets {
     pub num_local_memories: u32,
     /// The number of defined globals in the module.
     pub num_local_globals: u32,
+    /// Number of trap handlers in the module, 0 or 1 depending if signal-less mode is used.
+    pub num_trap_handlers: u32,
```

Comment:
This should be a boolean. I can't imaging why there will be more than one.

Something like `has_trap_handlers` seem like a much better option.

Reply:
Indeed, fixed.